### DISCUSSION TITLE
Trying to integrate rook-ceph with openstack-helm

### DISCUSSION THREAD

**Discussion by Alt-Shivam** [rook/rook#9953] [Q&A]
hey,
I am trying to integrate rook-ceph with openstack-helm
looking for some sort of help, How to execute this integration.

**Comment by BlaineEXE** [comment]
There is really a lot to cover here with such a broad question. I gave a talk on this a few years ago though. This should cover a lot of the basics. Feel free to follow up with more specific questions.

https://www.openstack.org/summit/denver-2019/summit-schedule/events/23237/rook-a-new-and-easy-way-to-run-your-ceph-storage-on-kubernetes

**Comment by Alt-Shivam** [comment]
Yeah, Sure
I think it's perfect for an initial start. :)
